CONTRIBUTING.md

```diff
@@ -80,6 +80,11 @@ There are two ways of running the `goat` locally:
     Note: Linter env variables must be formatted as <LINTER_CONFIG>, i.e. RUFF_CONFIG, CFN_LINT_CONFIG,
     etc., and the values supplied will overwrite the default arguments supplied in the goat.
 
+4. Autofix is available for certain linters and can be invoked in two ways:
```

Comment:
Remember, this is a GitHub action. Document the GitHub action method to do this. You will need to map an INPUT_AUTO_FIX to AUTO_FIX like we’ve done otherwise

Reply:
Created the map